upload/views/test_results.py

```diff
@@ -156,7 +158,7 @@ def post(self, request):
             "build_url": data.get("buildURL"),  # build_url
             "job": data.get("job"),  # job_code
             "flags": data.get("flags"),
-            "service": data.get("ci_service"),  # provider
+            "service": data.get("service"),  # git provider
             "url": storage_path,  # storage_path
             # these are used for dispatching the task below
             "commit": commit.commitid,
```

Comment:
@joseph-sentry can you double check the changes I made to the TA fields? Is this still the design we're going with?

Reply:
just confirmed, TA sends `git_service` as `service` https://github.com/codecov/codecov-cli/blob/2935526caf5db25d64b02870f9cc45e33a7daa3d/codecov_cli/commands/create_report_result.py#L36

these changes should be good to go